pymedphys/_mosaiq/helpers.py

```diff
@@ -319,3 +319,124 @@ def get_recently_completed_qcls_across_sites(
     results = results.sort_values(by="actual_completed_time", ascending=False)
 
     return results
+
+def get_mosaiq(mrn):
+    host_name = 'prwinvds006.utmsa.local'
+    with mosaiq_connect(host_name) as cursor:
```

Comment:
Here's an example of the testing suite doing its thing:

https://github.com/pymedphys/pymedphys/pull/541/checks?check_run_id=264817076#step:10:339

That's saying `mosaiq_connect` is undefined in this file, which caused that test to fail. You'll notice I don't actually import `mosaiq_connect` here, I let that be done by the user of the function and then pass a "cursor" object to the function instead:

https://github.com/pymedphys/pymedphys/blob/7ea86ff38bcb6c32deaee94a5be252ed3ce28f46/pymedphys/_mosaiq/helpers.py#L37